prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -183,3 +183,15 @@ func (o *KubernetesOptions) BuildClusterClients(namespace string, dryRun bool) (
 	}
 	return buildClients, nil
 }
+
+// BuildClusterClients returns K8S clients for build clusters.
+func (o *KubernetesOptions) BuildClusterK8SClients(dryRun bool) (buildClusterClients map[string]kubernetes.Interface, err error) {
```

Comment:
Why do we need to add a function for this? Don't we already have a way to access clients?

Nit: let's not use K8s - use Kubernetes

Reply:
The function above the defined is how `sindker/plank/...` gets hold of clients for `pod`:

https://github.com/kubernetes/test-infra/blob/345b76d20262a709fcfea8544036f5200af065b7/prow/flagutil/kubernetes_cluster_clients.go#L170-L171

I am doing similar things for configMap. The difference here is that namespace could be different for each config map.